app/authentication/auth.py

```diff
@@ -51,7 +62,9 @@ def requires_no_auth():
 def requires_admin_auth():
```

Comment:
`admin_auth` is for requests coming from the admin to the API. Some endpoints can only be called by admin frontends